cmd/apps_test.go

```diff
@@ -27,3 +37,35 @@ func TestExpandUrl(t *testing.T) {
 		}
 	}
 }
+
+func TestRemoteExists(t *testing.T) {
+	cf, server, err := testutil.NewTestServerAndClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.Close()
+
+	server.Mux.HandleFunc("/v2/apps/", func(w http.ResponseWriter, r *http.Request) {
+		testutil.SetHeaders(w)
+		fmt.Fprintf(w, `{
+    "owner": "jkirk",
+    "id": "foo",
+    "created": "2014-01-01T00:00:00UTC",
+    "updated": "2014-01-01T00:00:00UTC",
+    "uuid": "de1bf5b5-4a72-4f94-a10c-d2a3741cdf75"
+}`)
+	})
+
+	// create a remote first before running apps:create
+	assert.NoErr(t, git.Init())
```

Comment:
We run the tests in a container so it's wiped after each run. Are we to assume that these tests can be run out of the container?